keystore_test.go

```diff
@@ -246,68 +193,48 @@ func TestLoadKeyPassword(t *testing.T) {
 	defer zeroing(keyPassword)
 
 	f, err := os.Open("./testdata/keystore_keypass.jks")
-	if err != nil {
-		t.Fatalf("open test data keystore file: %s", err)
-	}
+	require.NoError(t, err)
 
 	defer func() {
-		if err := f.Close(); err != nil {
-			t.Fatalf("close test data keystore file: %s", err)
-		}
+		err := f.Close()
+		require.NoError(t, err)
 	}()
 
 	keyStore := New()
 
-	if err := keyStore.Load(f, password); err != nil {
-		t.Fatalf("decode test data keystore: %s", err)
-	}
+	err = keyStore.Load(f, password)
+	require.NoError(t, err)
 
 	actualPKE, err := keyStore.GetPrivateKeyEntry("alias", keyPassword)
-	if err != nil {
-		t.Fatalf("get private key entry: %s", err)
-	}
+	require.NoError(t, err)
 
 	expectedCT, err := time.Parse("2006-01-02 15:04:05.999999999 -0700 MST", "2020-10-26 12:01:38.387 +0200 EET")
-	if err != nil {
-		t.Fatalf("parse creation time: %s", err)
-	}
+	require.NoError(t, err)
 
-	if !actualPKE.CreationTime.Equal(expectedCT) {
-		t.Errorf("unexpected private key entry creation time: '%v' '%v'", actualPKE.CreationTime, expectedCT)
-	}
+	assert.Truef(t, actualPKE.CreationTime.Equal(expectedCT),
+		"unexpected private key entry creation time: '%v' '%v'", actualPKE.CreationTime, expectedCT)
 
-	if len(actualPKE.CertificateChain) != 1 {
-		t.Errorf("unexpected private key entry certificate chain length: '%d' '%d'", len(actualPKE.CertificateChain), 0)
-	}
+	assert.Lenf(t, actualPKE.CertificateChain, 1,
+		"unexpected private key entry certificate chain length: '%d' '%d'", len(actualPKE.CertificateChain), 0)
 
 	pkPEM, err := os.ReadFile("./testdata/key_keypass.pem")
-	if err != nil {
-		t.Fatalf("read expected private key file: %s", err)
-	}
+	require.NoError(t, err)
 
 	decodedPK, _ := pem.Decode(pkPEM)
 
-	if !reflect.DeepEqual(actualPKE.PrivateKey, decodedPK.Bytes) {
-		t.Errorf("unexpected private key %v \n %v", actualPKE.PrivateKey, decodedPK.Bytes)
-	}
+	assert.Truef(t, reflect.DeepEqual(actualPKE.PrivateKey, decodedPK.Bytes),
```

Comment:
Please consider using `assert.Equal` and put expected and actual arguments in a correct order.